frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts

```diff
@@ -554,4 +554,35 @@ export class ChallengeviewallsubmissionsComponent implements OnInit, AfterViewIn
     };
     SELF.globalService.showConfirm(PARAMS);
   }
+
+  /**
+   * Delete Submission.
+   * @param submission  Submission being deleted
+   */
+  SubmissionDelete(submission) {
```

Comment:
Can we please change the function name to `deleteChallengeSubmission` @Kajol-Kumari